ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.9.yaml

```diff
@@ -241,7 +241,7 @@ tests:
     workflow: openshift-e2e-libvirt
 - as: ocp-image-ecosystem-remote-libvirt-ppc64le
   cluster: build01
-  cron: 0 1,13 * * *
+  cron: 0 1 * * *
```

Comment:
We do not want to change anything for ppc64le

Reply:
addressed in the recent commit